melodies_monet/driver.py

```diff
@@ -1509,7 +1509,26 @@ def plotting(self):
 
                         # Query selected points if applicable
                         if domain_type != 'all':
-                            pairdf_all.query(domain_type + ' == ' + '"' + domain_name + '"', inplace=True)
+                            if "auto-region" in domain_type: 
+                                if 'epa_region' in domain_type:
+                                    bounds = get_epa_region_bounds(acronym=domain_name)
+                                elif 'giorgi_region' in domain_type:
```

Comment:
Alternative
```suggestion
                            if domain_type.startswith("auto-region"):
                                _, auto_region_type = domain_type.split(":")
                                if auto_region_type == 'epa':
                                    bounds = get_epa_region_bounds(acronym=domain_name)
                                elif auto_region_type == 'giorgi':
```
Also, to me, since "auto-region" already has "region" in it, the identifier doesn't need to.